arraycontainer.go

```diff
@@ -439,6 +439,7 @@ func (ac *arrayContainer) lazyorArray(value2 *arrayContainer) container {
 }
 
 func (ac *arrayContainer) and(a container) container {
+	//p("ac.and() called")
```

Comment:
@glycerine 

> I don't know what this would mean. They seem pretty self-evident.
 
I am sure you agree that coding is also very much a social activity where esthetic matters. For example, the Go language went with a very clean look.

I am not sure that these printouts are pretty. 

Now, there is an easy workaround. Just adopt a comment convention that is pretty and readable... then have a script turn these comments into printouts when you need them. Of course, this involves abiding by some standard way of writing these comments so that the script can do its work.

I don't think it is hard to do and I hope you'll agree that it is a reasonable compromise that is best on the long run. 

So instead of having...

```
	//p("ac.and() called")
```

We could have...

```
	/// ac.and() called
 ```
... or something... and then a script could turn it (temporarily) into 

```
	fmt.Println("ac.and() called")
 ```

See my point? 

I am sure we could improve on my proposal... but the idea is to keep the source code as pretty as possible, while still allowing you to have your printouts when you want to have them.